vmm/task/src/debug.rs

```diff
@@ -51,10 +52,23 @@ pub async fn listen_debug_console(addr: &str) -> Result<()> {
     Ok(())
 }
 
+pub async fn check_exec_cmd() -> Result<Command> {
+    let cmd = vec![
+        String::from("/usr/bin/bash"),
```

Comment:
Maybe we can make this a config, so that we don't need to add the path here if it is changed later.